jobs/flanneld/spec

```diff
@@ -20,6 +20,12 @@ properties:
   backend-type:
     description: The network backend to use
     default: "vxlan"
+  tls.etcdctl.ca:
+    description: CA for etcdctl client authentication
+  tls.etcdctl.certificate:
+    description: Certificate for etcdctl client authentication
+  tls.etcdctl.private_key:
+    description: Private key for etcdctl client authentication
 
 consumes:
 - name: etcd
```

Comment:
If I'm not mistaken, we can now remove the `consumes` link for `etcd`

Reply:
It still use etcd link for dns suffix
https://github.com/cloudfoundry-incubator/kubo-release/blob/4906d3a192584924a0ab85a25dafcc047af36a28/jobs/flanneld/templates/bin/flanneld_ctl.erb#L9

Reply:
And here https://github.com/cloudfoundry-incubator/kubo-release/blob/4906d3a192584924a0ab85a25dafcc047af36a28/jobs/flanneld/templates/bin/flanneld_ctl.erb#L39

Should we input them all as variable?